sympy/stats/crv_types.py

```diff
@@ -232,6 +232,17 @@ def Arcsin(name, a=0, b=1):
 class BeniniDistribution(SingleContinuousDistribution):
     _argnames = ('alpha', 'beta', 'sigma')
 
+    def __new__(name, *args):
+        alpha = args[BeniniDistribution._argnames.index('alpha')]
```

Comment:
Checking that there are exactly 3 arguments should be the first thing to do. Then it would suffice to write

    alpha, beta, sigma = args

Reply:
Ok, I'll do it. And also please tell me should I change some of the things which I mentioned above under **Other Comments**